docs/rest_api.rst

```diff
@@ -781,15 +781,15 @@ For ``raiden_events`` you can provide a ``limit`` and an ``offset`` number which
 
 Now for the internal Raiden events:
 
-.. http:get:: /api/(version)/_debug/raiden_events/tokens/(token_address)
+.. http:get:: /api/(version)/_debug/raiden_events/
```

Comment:
I think all `_debug` endpoints should be removed from the docs.

Reply:
Hmm it's good to mention at least which debug endpoints exist.

Reply:
why? I thought these end points where for debugging, IOW for us to test some stuff.

Reply:
For us and for any users we may need to ask to run these endpoints and provide us with the results. It's nice to have all the existing endpoints documented. The `_debug` endpoints could just have their existence documented for completeness sake and we can remove the examples.

I just want our docs to list all endpoints.